src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -330,31 +384,46 @@ bool emberAfOperationalCredentialsClusterRemoveAllFabricsCallback(chip::app::Com
     return true;
 }
 
-bool emberAfOperationalCredentialsClusterAddOpCertCallback(chip::app::Command * commandObj, chip::ByteSpan NOCArray,
-                                                           chip::ByteSpan IPKValue, chip::NodeId CaseAdminNode,
-                                                           uint16_t AdminVendorId)
+bool emberAfOperationalCredentialsClusterAddNOCCallback(chip::app::Command * commandObj, chip::ByteSpan NOCArray,
+                                                        chip::ByteSpan IPKValue, chip::NodeId CaseAdminNode, uint16_t AdminVendorId)
 {
-    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
+    CHIP_ERROR err                       = CHIP_NO_ERROR;
+    EmberAfNOCResponseStatus nocResponse = EMBER_ZCL_NOC_RESPONSE_STATUS_SUCCESS;
+    uint8_t fabricIndex                  = 0;
+    AdminPairingInfo * admin             = nullptr;
 
     emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: commissioner has added an Op Cert");
 
-    AdminPairingInfo * admin = retrieveCurrentAdmin();
-    VerifyOrExit(admin != nullptr, status = EMBER_ZCL_STATUS_FAILURE);
+    VerifyOrExit(CaseAdminNode != chip::kUndefinedNodeId && CaseAdminNode != chip::kAnyNodeId,
+                 nocResponse = EMBER_ZCL_NOC_RESPONSE_STATUS_MISSING_ACL);
```

Comment:
Please add a TODO that this is a workaround that must be removed when access control is implemented

Reply:
I think, `emberAfOperationalCredentialsClusterAddNOCCallback` will not even be called if the message is malformed (i.e. CaseAdminNode argument is missing). I don't know if we facility to return specific errors in the auto-generated code for such scenarios.

Any thoughts?

Reply:
> This error message must only arise if the CaseAdminNode argument is altogether missing in the command encoding.

If a required value is missing we will, in our current implementation, send a general "malformed command" response in generated code and never land in this code.

Reply:
OK, then it should be fine to not implement "missing ACL" for our stack implementation. Perhaps that error could even be removed from the spec, if IM should account for mandatory args errors?

Reply:
I'll remove it from the code. I agree, that it should also be removed from the spec.

Reply:
@tcarmelveilleux does the same apply for `MissingIpk` error?

Reply:
Also, hopefully we'll not update the values of other error codes after removing `MissingAcl` and `MissingIpk` from the spec. Otherwise, the code will go out of sync with the spec again. Or, I can assume that and adjust the values of the error codes...